src/AVFoundation/AVCompat.cs

```diff
@@ -10,6 +8,26 @@
 using ObjCRuntime;
 
 namespace AVFoundation {
+#if !XAMCORE_4_0
+	public delegate int AVAudioSourceNodeRenderHandler (bool isSilence, AudioToolbox.AudioTimeStamp timestamp, uint frameCount, ref AudioToolbox.AudioBuffers outputData);
```

Comment:
Is there a better way to make this delegate visible to the AVAudioSourceNode methods below? I tried `using AVFoundation.AVSourceNodeRenderHandler` but that failed compiling 😕